% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{article}
\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  pdftitle={KEGG pathway enrichment},
  pdfauthor={Camargo Escalante MO},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage[margin=1in]{geometry}
\usepackage{color}
\usepackage{fancyvrb}
\newcommand{\VerbBar}{|}
\newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\usepackage{framed}
\definecolor{shadecolor}{RGB}{248,248,248}
\newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
\newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\BuiltInTok}[1]{#1}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
\newcommand{\ExtensionTok}[1]{#1}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ImportTok}[1]{#1}
\newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\NormalTok}[1]{#1}
\newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
\newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\RegionMarkerTok}[1]{#1}
\newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi

\title{KEGG pathway enrichment}
\author{Camargo Escalante MO}
\date{2023-09-25}

\begin{document}
\maketitle

\hypertarget{kegg-pathway-enrichment-in-non-model-organisms-using-cluserprofiler}{%
\subsubsection{\texorpdfstring{KEGG pathway enrichment in non-model
organisms using
\emph{cluserProfiler}}{KEGG pathway enrichment in non-model organisms using cluserProfiler}}\label{kegg-pathway-enrichment-in-non-model-organisms-using-cluserprofiler}}

This document contains steps required to perform enrichment analysis
using the clusterprofiler package.

\hypertarget{how-is-structurated}{%
\paragraph{How is structurated:}\label{how-is-structurated}}

\minitoc

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{1) Install required packages}
\NormalTok{2) Load packages}
\NormalTok{3) Set the working and output directory}
\NormalTok{4) Create a table with IDs and ko:Knumber from input file 1}
\NormalTok{5) Perform KEGG pathway enrichment analysis}
\NormalTok{6) Plot results}
\end{Highlighting}
\end{Shaded}

For more details on using \emph{clusterprofiler} see
\url{http://yulab-smu.top/biomedical-knowledge-mining-book/index.html}.

\hypertarget{install-and-load-packages}{%
\paragraph{Install and load packages}\label{install-and-load-packages}}

Some packages can be installed from repository CRAN, others such
\href{https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html}{\emph{clusterprofiler}}
via Biocoductor. These lines are critical; if the enrichmente analysis
shows an error, it is necessary to rerun it.

Install and load \textbf{clusterprofiler} package and install
\textbf{R.utilis}:

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{library}\NormalTok{ (clusterProfiler)}
\FunctionTok{getOption}\NormalTok{(}\StringTok{"clusterProfiler.download.method"}\NormalTok{)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## [1] "libcurl"
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{\#install.packages("R.utils") \#Uncomment to install for the first time and reinstall if required (in case of error).}
\NormalTok{R.utils}\SpecialCharTok{::}\FunctionTok{setOption}\NormalTok{(}\StringTok{"clusterProfiler.download.method"}\NormalTok{,}\StringTok{"auto"}\NormalTok{)}
\end{Highlighting}
\end{Shaded}

Also install and load the next packages

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{library}\NormalTok{(}\StringTok{"topGO"}\NormalTok{)}
\FunctionTok{library}\NormalTok{(}\StringTok{"dplyr"}\NormalTok{)}
\FunctionTok{library}\NormalTok{(}\StringTok{"R.utils"}\NormalTok{)}
\FunctionTok{library}\NormalTok{(}\StringTok{"ggplot2"}\NormalTok{)}
\FunctionTok{library}\NormalTok{(}\StringTok{"enrichplot"}\NormalTok{)}
\FunctionTok{library}\NormalTok{(}\StringTok{"RColorBrewer"}\NormalTok{)}
\end{Highlighting}
\end{Shaded}

\hypertarget{set-working-directory-and-output-file-directory}{%
\paragraph{Set working directory and output file
directory}\label{set-working-directory-and-output-file-directory}}

The working directory is where the input files are:
\href{https://github.com/OrlanC/cp_enrichment}{Input file 1
(IDs-Knumber) and Input file 2 (IDs to be analyzed).}

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{setwd}\NormalTok{(}\StringTok{"C:/Users/Orlando Camargo/Desktop/cp\_enrichment/input\_files/"}\NormalTok{)}
\NormalTok{outpathcount }\OtherTok{=} \StringTok{"C:/Users/Orlando Camargo/Desktop/cp\_enrichment/results/"}
\end{Highlighting}
\end{Shaded}

\hypertarget{create-a-table-with-ids-and-koknumber-from-input-file-1}{%
\paragraph{Create a table with IDs and ko:Knumber from input file
1}\label{create-a-table-with-ids-and-koknumber-from-input-file-1}}

It is important to generate a organized table where each Knumber has its
corresponding ID.

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{KEGG\_db}\OtherTok{\textless{}{-}} \FunctionTok{readMappings}\NormalTok{(}\StringTok{"Tatroviride\_protein\_KEGG\_ko.txt"}\NormalTok{,}\AttributeTok{sep=}\StringTok{"}\SpecialCharTok{\textbackslash{}t}\StringTok{"}\NormalTok{, }\AttributeTok{IDsep =} \StringTok{","}\NormalTok{)}

\NormalTok{KEGGpath}\OtherTok{\textless{}{-}}\FunctionTok{data.frame}\NormalTok{()}
\ControlFlowTok{for}\NormalTok{(i }\ControlFlowTok{in} \FunctionTok{names}\NormalTok{(KEGG\_db))\{}
\NormalTok{  y }\OtherTok{=}\NormalTok{KEGG\_db[[i]]}
\NormalTok{  ln}\OtherTok{\textless{}{-}}\FunctionTok{length}\NormalTok{(KEGG\_db[[i]])}
\NormalTok{  x}\OtherTok{=}\FunctionTok{data.frame}\NormalTok{(}\AttributeTok{ID=}\FunctionTok{rep}\NormalTok{(i,ln),y)}
\NormalTok{  KEGGpath}\OtherTok{\textless{}{-}}\FunctionTok{rbind}\NormalTok{(KEGGpath,x)}
\NormalTok{\}}
\NormalTok{KEGGpath}\SpecialCharTok{$}\NormalTok{y}\OtherTok{\textless{}{-}}\FunctionTok{sub}\NormalTok{(}\StringTok{"ko:"}\NormalTok{,}\StringTok{""}\NormalTok{,KEGGpath}\SpecialCharTok{$}\NormalTok{y)}
\FunctionTok{colnames}\NormalTok{(KEGGpath)}\OtherTok{\textless{}{-}}\FunctionTok{c}\NormalTok{(}\StringTok{"ID"}\NormalTok{,}\StringTok{"KEGG"}\NormalTok{)}

\NormalTok{KEGGpath[}\DecValTok{60}\SpecialCharTok{:}\DecValTok{80}\NormalTok{,]}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
##                 ID   KEGG
## 60 Tatro_000067-T1      -
## 61 Tatro_000069-T1      -
## 62 Tatro_000070-T1      -
## 63 Tatro_000071-T1      -
## 64 Tatro_000074-T1      -
## 65 Tatro_000075-T1 K02919
## 66 Tatro_000076-T1 K20794
## 67 Tatro_000077-T1 K00088
## 68 Tatro_000078-T1 K20178
## 69 Tatro_000079-T1 K03861
## 70 Tatro_000080-T1 K14779
## 71 Tatro_000081-T1 K04567
## 72 Tatro_000082-T1 K11340
## 73 Tatro_000082-T1 K11400
## 74 Tatro_000082-T1 K11652
## 75 Tatro_000083-T1      -
## 76 Tatro_000084-T1 K17732
## 77 Tatro_000085-T1 K03133
## 78 Tatro_000086-T1 K14152
## 79 Tatro_000087-T1 K15728
## 80 Tatro_000087-T2 K15728
\end{verbatim}

Read input file 2, this can be the differential expression result table
or a ID list in .txt.

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{data }\OtherTok{=} \FunctionTok{read.table}\NormalTok{(}\StringTok{"input\_file\_2.txt"}\NormalTok{, }\AttributeTok{header =} \ConstantTok{TRUE}\NormalTok{, }\AttributeTok{sep=}\StringTok{"}\SpecialCharTok{\textbackslash{}t}\StringTok{"}\NormalTok{,  }\AttributeTok{quote=} \StringTok{""}\NormalTok{, }\AttributeTok{row.names=}\DecValTok{1}\NormalTok{, }\AttributeTok{comment.char=}\StringTok{""}\NormalTok{)}
\FunctionTok{head}\NormalTok{(data)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
##                     logFC   logCPM        F       PValue          FDR
## Tatro_002208-T1 -1.843287 5.400751 97.03844 1.756688e-09 5.520017e-06
## Tatro_011096-T1 -1.203280 5.524546 95.27822 2.069822e-09 5.520017e-06
## Tatro_008592-T1 -1.540310 6.419002 93.47258 2.455524e-09 5.520017e-06
## Tatro_011193-T1 -1.776928 5.592757 87.86123 4.251203e-09 7.645363e-06
## Tatro_007063-T1 -1.390135 6.720424 70.98038 2.679955e-08 1.668623e-05
## Tatro_003659-T1 -1.955451 4.865080 70.74980 2.754515e-08 1.668623e-05
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{id}\OtherTok{\textless{}{-}}\FunctionTok{rownames}\NormalTok{(data)}
\NormalTok{id\_ko}\OtherTok{\textless{}{-}}\NormalTok{KEGGpath[}\FunctionTok{which}\NormalTok{(KEGGpath}\SpecialCharTok{$}\NormalTok{ID}\SpecialCharTok{\%in\%}\NormalTok{id),]}
\FunctionTok{head}\NormalTok{(id\_ko, }\AttributeTok{n =} \DecValTok{10}\NormalTok{)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
##                  ID   KEGG
## 2   Tatro_000001-T1      -
## 3   Tatro_000002-T1      -
## 21  Tatro_000022-T1      -
## 85  Tatro_000092-T1 K00972
## 203 Tatro_000219-T1      -
## 214 Tatro_000233-T1 K07300
## 225 Tatro_000243-T1 K13621
## 268 Tatro_000287-T1 K16055
## 279 Tatro_000299-T1      -
## 287 Tatro_000311-T1      -
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{length}\NormalTok{(id\_ko}\SpecialCharTok{$}\NormalTok{ID)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## [1] 360
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{ko}\OtherTok{\textless{}{-}}\NormalTok{id\_ko[id\_ko}\SpecialCharTok{$}\NormalTok{KEGG}\SpecialCharTok{!=}\StringTok{"{-}"}\NormalTok{,] }\CommentTok{\# Delete those IDs that have non{-}Knumber associated.}
\FunctionTok{head}\NormalTok{(ko)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
##                  ID   KEGG
## 85  Tatro_000092-T1 K00972
## 214 Tatro_000233-T1 K07300
## 225 Tatro_000243-T1 K13621
## 268 Tatro_000287-T1 K16055
## 321 Tatro_000347-T1 K08197
## 328 Tatro_000355-T1 K03381
\end{verbatim}

\hypertarget{perform-the-kegg-pathway-enrichment-analysis}{%
\paragraph{Perform the KEGG pathway enrichment
analysis}\label{perform-the-kegg-pathway-enrichment-analysis}}

Use Knumber from data frame \textbf{ko}. In this step, you might get an
error; if it happens, go back to \textbf{Install and load packages}.

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{knum}\OtherTok{\textless{}{-}}\NormalTok{ko}\SpecialCharTok{$}\NormalTok{KEGG}

\NormalTok{enKEGG}\OtherTok{\textless{}{-}}\FunctionTok{enrichKEGG}\NormalTok{(knum, }\AttributeTok{organism =} \StringTok{\textquotesingle{}ko\textquotesingle{}}\NormalTok{, }\AttributeTok{minGSSize =} \DecValTok{1}\NormalTok{,}\AttributeTok{keyType =} \StringTok{"kegg"}\NormalTok{, }\AttributeTok{pvalueCutoff =} \FloatTok{0.05}\NormalTok{,}\AttributeTok{pAdjustMethod =} \StringTok{"BH"}\NormalTok{, }\AttributeTok{qvalueCutoff =} \DecValTok{1}\NormalTok{)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## Reading KEGG annotation online:
## 
## Reading KEGG annotation online:
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{write.table}\NormalTok{(enKEGG, }\AttributeTok{file=}\StringTok{"KEGG.txt"}\NormalTok{, }\AttributeTok{row.names=}\ConstantTok{TRUE}\NormalTok{, }\AttributeTok{col.names=}\ConstantTok{NA}\NormalTok{, }\AttributeTok{quote=}\ConstantTok{FALSE}\NormalTok{, }\AttributeTok{sep=}\StringTok{"}\SpecialCharTok{\textbackslash{}t}\StringTok{"}\NormalTok{) }\CommentTok{\#export only significant results, use enKEGG@result to get all results (significant and not significant results)}

\NormalTok{enKEGG}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## #
## # over-representation test
## #
## #...@organism     ko 
## #...@ontology     KEGG 
## #...@keytype      kegg 
## #...@gene     chr [1:141] "K00972" "K07300" "K13621" "K16055" "K08197" "K03381" "K00480" ...
## #...pvalues adjusted by 'BH' with cutoff <0.05 
## #...16 enriched terms found
## 'data.frame':    16 obs. of  9 variables:
##  $ ID         : chr  "map00564" "map00500" "map04814" "map01212" ...
##  $ Description: chr  "Glycerophospholipid metabolism" "Starch and sucrose metabolism" "Motor proteins" "Fatty acid metabolism" ...
##  $ GeneRatio  : chr  "9/93" "7/93" "7/93" "6/93" ...
##  $ BgRatio    : chr  "115/13850" "106/13850" "113/13850" "84/13850" ...
##  $ pvalue     : num  7.34e-08 6.95e-06 1.06e-05 2.08e-05 3.08e-05 ...
##  $ p.adjust   : num  0.000012 0.00057 0.00058 0.000852 0.001011 ...
##  $ qvalue     : num  1.01e-05 4.79e-04 4.88e-04 7.16e-04 8.50e-04 ...
##  $ geneID     : chr  "K13621/K01126/K13507/K18696/K16369/K00006/K00111/K01114/K01115" "K16055/K05349/K01835/K01194/K01196/K00963/K01179" "K10426/K07374/K10413/K10396/K10401/K10392/K10357" "K00626/K00667/K00668/K00507/K11262/K10256" ...
##  $ Count      : int  9 7 7 6 9 7 6 3 3 3 ...
## #...Citation
##  T Wu, E Hu, S Xu, M Chen, P Guo, Z Dai, T Feng, L Zhou, W Tang, L Zhan, X Fu, S Liu, X Bo, and G Yu.
##  clusterProfiler 4.0: A universal enrichment tool for interpreting omics data.
##  The Innovation. 2021, 2(3):100141
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{head}\NormalTok{(enKEGG}\SpecialCharTok{@}\NormalTok{result)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
##                ID                                 Description GeneRatio
## map00564 map00564              Glycerophospholipid metabolism      9/93
## map00500 map00500               Starch and sucrose metabolism      7/93
## map04814 map04814                              Motor proteins      7/93
## map01212 map01212                       Fatty acid metabolism      6/93
## map01230 map01230                 Biosynthesis of amino acids      9/93
## map00520 map00520 Amino sugar and nucleotide sugar metabolism      7/93
##            BgRatio       pvalue     p.adjust       qvalue
## map00564 115/13850 7.341551e-08 1.204014e-05 1.012361e-05
## map00500 106/13850 6.954338e-06 5.702557e-04 4.794833e-04
## map04814 113/13850 1.060942e-05 5.799814e-04 4.876609e-04
## map01212  84/13850 2.078043e-05 8.519977e-04 7.163780e-04
## map01230 238/13850 3.082017e-05 1.010902e-03 8.499878e-04
## map00520 156/13850 8.464802e-05 2.014599e-03 1.693918e-03
##                                                                  geneID Count
## map00564 K13621/K01126/K13507/K18696/K16369/K00006/K00111/K01114/K01115     9
## map00500               K16055/K05349/K01835/K01194/K01196/K00963/K01179     7
## map04814               K10426/K07374/K10413/K10396/K10401/K10392/K10357     7
## map01212                      K00626/K00667/K00668/K00507/K11262/K10256     6
## map01230 K01754/K13830/K01702/K01850/K00838/K00549/K00789/K00826/K03785     9
## map00520               K00972/K20844/K01183/K12373/K01835/K00963/K00820     7
\end{verbatim}

\hypertarget{plot-results}{%
\paragraph{Plot results}\label{plot-results}}

With \emph{enrichplot} it is possible generate distinct plots to
represent enrichment results, either using all the significant results
or some. Also, with \emph{clusterprofiler}, the results can be
visualized in a KEGG pathway with a web browser.

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{p1}\OtherTok{\textless{}{-}}\FunctionTok{dotplot}\NormalTok{(enKEGG)}
\NormalTok{p2}\OtherTok{\textless{}{-}}\FunctionTok{barplot}\NormalTok{(enKEGG)}
\NormalTok{cowplot}\SpecialCharTok{::}\FunctionTok{plot\_grid}\NormalTok{(p1, p2,}\AttributeTok{ncol=}\DecValTok{2}\NormalTok{, }\AttributeTok{labels=}\NormalTok{LETTERS[}\DecValTok{1}\SpecialCharTok{:}\DecValTok{2}\NormalTok{], }\AttributeTok{rel\_widths=}\FunctionTok{c}\NormalTok{(}\DecValTok{2}\NormalTok{, }\DecValTok{2}\NormalTok{))}
\end{Highlighting}
\end{Shaded}

\includegraphics{kegg_pathway_enrichment_files/figure-latex/plot-1.pdf}

\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{\#browseKEGG(enKEGG, map00564)}
\end{Highlighting}
\end{Shaded}


\end{document}
